test/topology_experimental_raft/test_tablets.py

```diff
@@ -646,3 +646,114 @@ async def check():
+
+    # When one of tables is dropped
+    await cql.run_async("DROP TABLE testing.mytable2;")
+
+    # Then tablet count metric for each shard is 2
+    expected_count_per_shard = [
+        {"shard_id": 0, "tablet_count": 2},
+        {"shard_id": 1, "tablet_count": 2},
+        {"shard_id": 2, "tablet_count": 2},
+        {"shard_id": 3, "tablet_count": 2},
+    ]
+    await assert_tablet_count_metric_value_for_shards(manager, servers[0], expected_count_per_shard)
+    await assert_tablet_count_metric_value_for_shards(manager, servers[1], expected_count_per_shard)
+
+    # When disabling load balancing
+    await manager.api.disable_tablet_balancing(servers[0].ip_addr)
+    await manager.api.disable_tablet_balancing(servers[1].ip_addr)
+
+    # And when moving tablets from (host_0, shard_0) to (host_1, shard_3)
+    host0_id = await manager.get_host_id(servers[0].server_id)
+    host1_id = await manager.get_host_id(servers[1].server_id)
+
+    table1_tablets = await get_all_tablet_replicas(manager, servers[0], "testing", "mytable1")
+    table1_token = get_last_tablet_token_for_shard(table1_tablets, host0_id, 0)
+
+    table3_tablets = await get_all_tablet_replicas(manager, servers[0], "testing", "mytable3")
+    table3_token = get_last_tablet_token_for_shard(table3_tablets, host0_id, 0)
+
+    await manager.api.move_tablet(node_ip=servers[0].ip_addr, ks="testing", table="mytable1", src_host=host0_id, src_shard=0, dst_host=host1_id, dst_shard=3, token=table1_token)
+    await manager.api.move_tablet(node_ip=servers[0].ip_addr, ks="testing", table="mytable3", src_host=host0_id, src_shard=0, dst_host=host1_id, dst_shard=3, token=table3_token)
+
+    # And when ensuring that local tablet metadata on the queried node reflects the finalized tablet movement
+    host0 = manager.get_cql().cluster.metadata.get_host(servers[0].ip_addr)
+    host1 = manager.get_cql().cluster.metadata.get_host(servers[1].ip_addr)
+    await read_barrier(manager.get_cql(), host0)
+    await read_barrier(manager.get_cql(), host1)
+
+    # Then tablet count metric is adjusted to depict that situation on host0
+    expected_count_per_shard = [
+        {"shard_id": 0, "tablet_count": 0},
+        {"shard_id": 1, "tablet_count": 2},
+        {"shard_id": 2, "tablet_count": 2},
+        {"shard_id": 3, "tablet_count": 2},
+    ]
+    await assert_tablet_count_metric_value_for_shards(manager, servers[0], expected_count_per_shard)
+
+    # And then tablet count metric is increased on host1
+    expected_count_per_shard = [
+        {"shard_id": 0, "tablet_count": 2},
+        {"shard_id": 1, "tablet_count": 2},
+        {"shard_id": 2, "tablet_count": 2},
+        {"shard_id": 3, "tablet_count": 4},
+    ]
+    await assert_tablet_count_metric_value_for_shards(manager, servers[1], expected_count_per_shard)
+
+    # And manager stops servers gracefully
+    await manager.server_stop_gracefully(servers[0].server_id, timeout=120)
```

Comment:
Better not to do it, the test takes more time.

Reply:
Okay, sure. I was not aware of it. Fixed :+1:

Reply:
@pwrobelse - if you've fixed this issue - please resolve conversation. Same for all other areas you've responded to. Makes understanding the state of things here easier.